app/src/main/res/layout/view_browser_navigation_bar.xml

```diff
@@ -82,20 +83,15 @@
         android:layout_height="match_parent"
         android:layout_weight="1" />
 
-    <FrameLayout
+    <com.duckduckgo.app.browser.tabswitcher.ExperimentalTabSwitcherButton
         android:id="@+id/tabsButton"
-        android:layout_width="40dp"
-        android:layout_height="40dp"
-        android:background="?attr/selectableItemBackgroundBorderless">
-
-        <ImageView
-            android:id="@+id/tabsIconImageView"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:layout_gravity="center"
-            android:padding="@dimen/keyline_2"
-            android:src="@drawable/ic_tab_24e" />
```

Comment:
Could you also remove the tabs image resource?

Reply:
will do!